api/Window.json

```diff
@@ -4418,6 +4559,55 @@
           }
         }
       },
+      "navigate": {
```

Comment:
I don't think we should add this entry, as it's not in any recent browser release.